source/includes/quick-start/overview.rst

```diff
@@ -0,0 +1,14 @@
+This guide shows you how to create an application that uses the **Java driver**
+to connect to a **MongoDB Atlas cluster**. If you prefer to connect to MongoDB
+using a different driver or programming language, see our
+:driver:`list of official MongoDB drivers <>`.
+
+The Java driver is an interface through which you can connect to and
```

Comment:
It's an interface, yes, but it's the implementation of the interface that allows you to connect.  Maybe just say:

        The Java driver lets you connect to and communicate with ... from a Java application.

or something to that effect